bundles/admin/admin-layereditor/view/AdminLayerForm/VisualizationTabPane/Scale.jsx

```diff
@@ -78,7 +78,7 @@ const Scale = ({ layer, scales = [], controller, getMessage }) => {
     useEffect(() => {
         if (layer && !layerChecked) {
             setLayerChecked(true);
-            const {minscale, maxscale} = layer;
+            const {minscale = 1, maxscale = -1} = layer;
```

Comment:
is minScale 1 and not -1 on purpose here?

Reply:
no. made some other changes and reverted them with this mistake. fixed.